flexmeasures/data/scripts/data_gen.py

```diff
@@ -303,9 +303,9 @@ def depopulate_prognoses(
         % db.engine
     )
 
-    # Clear all jobs
-    num_forecasting_jobs_deleted = app.queues["forecasting"].empty()
-    num_scheduling_jobs_deleted = app.queues["scheduling"].empty()
+    if not sensor_id:
```

Comment:
The function could use a docstring note explaining the difference in behaviour when passing a sensor id. Specifically, when a sensor id is passed, there may still remain jobs in the queues for this sensor. That may even be worth a `click.echo` under an `else` block.